packages/theme/src/i18n/en/openDialog.ts

```diff
@@ -32,5 +32,5 @@ export const openDialog = {
   startCollaboratingDescription:
     "Make the most of your Foxglove account – whether you want to dive deep on your data or share tools with your teammates.",
```

Comment:
Can you please change to
`"Make the most of your Foxbox account – whether you want to dive deep on your data or share tools with your teammates."` ?